packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2060,7 +2071,8 @@ export default class Meeting extends StatelessWebexPlugin {
    * @returns {void}
    */
   handleDataChannelUrlChange(datachannelUrl) {
-    if (datachannelUrl && this.config.enableAutomaticLLM) {
+    // @ts-ignore - config coming from registerPlugin
```

Comment:
why did you add this?

Reply:
cause this.config was complaining; so I did what is done in this file about 20 times, like in line :2544